src/lighteval/metrics/metrics_sample.py

```diff
@@ -616,3 +618,58 @@ def edit_similarity(self, s1, s2):
         """
         edist = edit_distance(s1, s2)
         return 1.0 - edist / max(len(s1), len(s2)) if len(s1) > 0 and len(s2) > 0 else 0
+
+
+class JudgeLLM:
+    available_models = ["gpt-3.5-turbo"]
```

Comment:
yes ! the goal is to have some closed source and if you pass a model like "org/model" it will load a model from HF. will open another PR for this though

Reply:
perfect, don't forget to create an issue for it!